.github/workflows/CI.yml

```diff
@@ -202,12 +202,40 @@ jobs:
         name: wheels
         path: dist
 
-  pypy:
-    runs-on: ${{ matrix.os }}
+  pypy-linux:
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ pypy-3.6, pypy-3.7, pypy-3.8 ]
```

Comment:
Drop pypy 3.6 support since it's not actively developed and manylinux docker images only have pypy 3.7 and 3.8 now.